tests/fixtures/typescript/src/test01.ts

```diff
@@ -7,6 +7,10 @@ export type Emits1 = {
   (e: 'foo' | 'bar', payload: string): void
   (e: 'baz', payload: number): void
 }
+export type Emits2 = {
+  (e: 'click' | 'bar', payload: string): void
+  (e: 'keydown', payload: number): void
+}
```

Comment:
What is this used for?

Reply:
This is for the test on [line 1108](https://github.com/JoCa96/eslint-plugin-vue/blob/df3e87ee5257002ac1d3e6ea6b0203da2ef00712/tests/lib/rules/no-shadow-native-events.js#L1108) which ensures that imported types are also considered:

```vue
<script setup lang="ts">
  import {Emits2 as Emits} from './test01'
  const emit = defineEmits<Emits>()
</script>
```